src/views/about.njk

```diff
@@ -2,6 +2,11 @@
 
 {% block content %}
 <main>
+  <header class="main-header">
+    <div class="main-title">
+      <h1>About The Federation</h1>
```

Comment:
The federation is the title of the home page. This is for the "About" page. But yeah I'm open to something better.